phpunit.xml.dist

```diff
@@ -2,6 +2,10 @@
 
 <phpunit bootstrap="./Tests/bootstrap.php" colors="true">
 
+    <php>
+        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[direct]=0" />
```

Comment:
How about using `weak` as value?

Reply:
We are using the latest version of the bridge. There is no "weak" anymore I believe?

https://symfony.com/doc/current/components/phpunit_bridge.html#making-tests-fail

Reply:
I see. Good to know :)